frontend/packages/pipelines-plugin/src/utils/pipeline-filter-reducer.ts

```diff
@@ -97,3 +146,8 @@ export const taskRunFilterReducer = (taskRun): string => {
   const status = pipelineRunStatus(taskRun);
```

Comment:
Same here,`pipelineFilterReducer` returns an enum, so `taskRunFilterReducer` should also return an enum, not. a string.